client/src/components/BookingButton.jsx

```diff
@@ -6,16 +6,20 @@ class BookingButton extends React.Component {
   }
 
   render() {
-    if (this.props.instantBook) {
+    if (this.props.bookingType) {
```

Comment:
use ternary operators here 

```react.js
{ 
  this.props.bookingType ? (
    <OneComponent />
  ) : (
    <AnotherComponent />
  )
}
```

Reply:
Is it like implicit return while using arrow funcs?